integration-tests/terraform/common/output.tf

```diff
@@ -0,0 +1,23 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+output "testing_id" {
```

Comment:
Do we need any of these outputs? I dont see anything using them.

Reply:
Using this in generating unique cluster creation and iam role creation.


resource "aws_eks_cluster" "this" {
  name     = "cwagent-operator-eks-integ-${module.common.testing_id}"
  role_arn = module.basic_components.role_arn
  version  = var.k8s_version
  vpc_config {
    subnet_ids         = module.basic_components.public_subnet_ids
    security_group_ids = [module.basic_components.security_group]
  }
}